tests/libinfra/certificates.go

```diff
@@ -40,3 +49,40 @@ func ContainsCrt(bundle []byte, containedCrt []byte) bool {
 	}
 	return attached
 }
+
+func GetBundleFromConfigMap(ctx context.Context, configMapName string) ([]byte, []*x509.Certificate) {
```

Comment:
This function is called 3 time. In two of them, we only use the first return value, and in the other one we only use the second return value, but never both of them. Also, these two values are, in a way, the same data, with different package.

I wander if we need to only return the byte slice, and do the certification logic in the one place it's needed.